specifications/architecture/README.md

```diff
@@ -0,0 +1,89 @@
+# blindnet devkit Architecture
+
+This document specifies the architecture of the blindnet devkit.
+
+> **Note**
+>
+> The current version of this document is based mainly on the [HLA][HLA]. It is expected for this document to evolve, especially once the [functional requirements document](../../specifications#functional-requirements) is available.
+
+## Terminology
+
+- All terms defined in [RFC-Lexicon-2][Lexicon] are to be interpreted as described there
+- Any additional precision about the terms defined in [RFC-Lexicon-2][Lexicon], as well as additional terms such as Consent and Legal Base, provided in [High Level Conceptualization][HLC] is to be considered normative
+- We use the terms Capture Component, Encryption and Access Management Engine, Privacy Computation Engine, Privacy Compiler, Privacy Request Capture Interface, Schemas and Storage Component as defined in [High Level Architecture][HLA]
+
+## Responsibilities
+
+Figure below presents the blindnet devkit architecture.
+
+<img src="./img/devkit_architecture.png">
+<br><br>
+
+Each element within the architecture is responsible for a certain set of functions within the blindnet devkit.
+
+**Privacy Request Capture Interface**, which is a part of **Privacy Request Manager** as described in [HLA][HLA], is an end-user interface allowing Data Subjects to submit Privacy Requests. For requests that require it, this component may also initiate Data Subject Authentication, deliver the Privacy Request Response (including the data that may be part of response) and show its status.
+
+**Data Consumer Interface**, which is a part of **Privacy Request Manager** as described in [HLA][HLA], is an end-user interface for Data Consumers which allows them to:
+
+*Q3 2022 Scope*:
+- Set relevant configurations (e.g., Retention Policies, General Information for TRANSPARENCY requests, Transfers information, desired level of automation of Privacy Request Processing )
+- Manage Privacy Requests (View Privacy Requests, Act upon Privacy Request Response recommendations - grant/deny/transfer requests)
+
+*Autumn 2022 Scope*:
+- View and manage Data Captures
+
+**Web components** are front-end, look and feel agnostic components which allow integrations of different blindnet devkit functions into external systems and web sites. Currently, these include:
+- Login component (System must be able to work with existing login engines of client Systems)
```

Comment:
Can we find a way to not make the login component ourselves at all, but fully rely on client's login solution or an open-source component?